Gemfile

```diff
@@ -19,9 +19,9 @@ gem "sdoc", "~> 0.4.0", group: :doc
 gem "pg", "~> 0.20.0"
 gem "dotenv-rails"
 
-gem "devise",           "~> 3.5.2"
-gem "devise_invitable", "~> 1.5.2"
-gem "devise_security_extension"
+gem "devise",           "~> 4.6.1"
+gem "devise_invitable", "~> 1.7.5"
+gem 'devise_security_extension', github: 'phatworx/devise_security_extension'
```

Comment:
Why are we switching to use an unreleased version here?

Reply:
The current release removed `password_salt` from `OldPassword`. This branch resolves that.

Reply:
Happy to use the main repo, once we've unblocked, but for now I decided to use this branch to limit the amount of change going on at the moment.

Reply:
We need to move to a maintained version of this gem: https://github.com/devise-security/devise-security - Adding a ticket for future work to fix this up.